Parser/asdl_c.py

```diff
@@ -961,8 +990,12 @@ def visitProduct(self, prod, name):
         self.emit_defaults(name, prod.attributes, 1)
 
     def visitSum(self, sum, name):
-        self.emit('state->%s_type = make_type("%s", state->AST_type, NULL, 0);' %
-                  (name, name), 1)
+        self.emit(
+            'state->%s_type = make_type("%s", state->AST_type, NULL, 0, %s);'
+            % (name, name, reflow_c_string(asdl_of(name, sum), 1)),
+            1,
+            reflow=False,
+        )
```

Comment:
If the docstring is single-line, it may be too long. And if it is multi-line, the first line is not aligned with the rest. I suggest to break a line before docstring.
```suggestion
        self.emit('state->%s_type = make_type("%s", state->AST_type, NULL, 0,' %
                  (name, name), 1)
        self.emit('%s);' % reflow_c_string(asdl_of(name, sum), 2), 2, reflow=False)
```